mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -466,6 +466,8 @@
     <string name="wallpaper_tap_to_change_switch_label">Tap logo to change wallpaper</string>
     <!-- Description for switch which toggles the "tap-to-switch" behavior on home screen logo -->
     <string name="wallpaper_tap_to_change_switch_description">Cycle through and update image without leaving the homepage.</string>
+    <!-- Content description for switch which toggles the "tap-to-switch" behavior on home screen logo -->
+    <string name="wallpaper_logo_content_description">Firefox logo - change the wallpaper, button</string>
```

Comment:
This string looks weird to me (espcieally the "button" part in the string) - I can't understand what it means even looking at the comment
Could you please help explain more? thank you :)

Reply:
Sorry, let help to clarify.
This is for the new wallpapers functionality, as part of the feature users could tap the Firefox logo in the home screen and can switch to different wallpapers by tapping. The text was provided by the product team on https://mozilla-hub.atlassian.net/browse/FNXV2-19315 we could suggest any improvements if it's needed ? :)

Reply:
Oh! of course, it's an accessibility string! Now everything makes sense :P 
Could you please call that out explicitly in comments?
thank you!

Reply:
Sure, I'll update the PR :)
Thanks for all the help!

Reply:
I opened https://github.com/mozilla-mobile/fenix/pull/23562 for it, please let me know what do think :)

Reply:
Perfect! thanks :)